### PR TITLE
DOC: Remove docstring examples

### PR DESCRIPTION
**Pull request by rrjbca**
## Description
Removes all docstring examples. Currently they only demonstrate trivial call patterns which should be adequately described by the list of parameters in the docstrings for each function. Any more complex examples that are required should probably be added to the examples directory instead where text, plots and code examples can be better used to explain and demonstrate what the function is doing. Docstring examples are also currently run as part of the tests which is unnecessary given that the actual unit tests should all cover the same simple call patterns.

## Checklist
- [x] Follow the [Contributor Guidelines](https://github.com/skypyproject/skypy/blob/master/CONTRIBUTING.md)
- [ ] Write unit tests
- [x] Write documentation strings
- [ ] Assign someone from your working team to review this pull request
- [x] Assign someone from the infrastructure team to review this pull request

### CODE REVIEW

File: skypy/galaxies/luminosity.py [hunk 1]
```diff
@@ -46,18 +46,6 @@ def schechter_lf_magnitude(redshift, M_star, alpha, m_lim, cosmology, size=None,
         Absolute magnitude sampled from a Schechter luminosity function for
         each input galaxy redshift.
```
Comment (Lucia-Fonseca) [inline skypy/galaxies/luminosity.py:48]: Do we need this blank line?
```suggestion
```

File: skypy/galaxies/morphology.py [hunk 2]
```diff
@@ -89,14 +82,6 @@ def beta_ellipticity(e_ratio, e_sum, size=None):
     ----------
     .. [1] Kacprzak T., Herbel J., Nicola A. et al., arXiv:1906.01018
```
Comment (Lucia-Fonseca) [inline skypy/galaxies/morphology.py:84]:
```suggestion
```

File: skypy/galaxies/morphology.py [hunk 3]
```diff
@@ -300,15 +255,6 @@ def ryden04_ellipticity(mu_gamma, sigma_gamma, mu, sigma, size=None):
     ----------
     .. [1] Ryden B. S., 2004, ApJ, 601, 214
```
Comment (Lucia-Fonseca) [inline skypy/galaxies/morphology.py:257]:
```suggestion
```

File: skypy/galaxies/redshift.py [hunk 4]
```diff
@@ -58,14 +58,6 @@ def smail(z_median, alpha, beta, size=None):
     .. [1] Smail I., Ellis R. S., Fitchett M. J., 1994, MNRAS, 270, 245
     .. [2] Amara A., Refregier A., 2007, MNRAS, 381, 1018
```
Comment (Lucia-Fonseca) [inline skypy/galaxies/redshift.py:60]:
```suggestion
```

File: skypy/galaxies/redshift.py [hunk 5]
```diff
@@ -117,23 +109,6 @@ def schechter_lf_redshift(redshift, M_star, phi_star, alpha, m_lim, sky_area,
         Redshifts of the galaxy sample described by the Schechter luminosity
         function.
```
Comment (Lucia-Fonseca) [inline skypy/galaxies/redshift.py:111]:
```suggestion
```

File: skypy/galaxies/redshift.py [hunk 6]
```diff
@@ -260,18 +235,6 @@ def redshifts_from_comoving_density(redshift, density, sky_area, cosmology, nois
         Sampled redshifts such that the comoving number density of galaxies
         corresponds to the input distribution.
```
Comment (Lucia-Fonseca) [inline skypy/galaxies/redshift.py:237]:
```suggestion
```

File: skypy/galaxies/spectrum.py [hunk 7]
```diff
@@ -73,20 +73,6 @@ def dirichlet_coefficients(redshift, alpha0, alpha1, z1=1., weight=None):
     .. [2] Blanton M. R., Roweis S., 2007, The Astronomical Journal,
            Volume 133, Page 734
```
Comment (Lucia-Fonseca) [inline skypy/galaxies/spectrum.py:75]:
```suggestion
```

File: skypy/utils/special.py [hunk 8]
```diff
@@ -70,15 +70,6 @@ def gammaincc(a, x):
     --------
     scipy.special.gammaincc : Computes the start of the recurrence.
```
Comment (Lucia-Fonseca) [inline skypy/utils/special.py:72]:
```suggestion
```

**Review by Lucia-Fonseca — CHANGES_REQUESTED** [review]
Delete blank lines

**Comment by ntessore** [comment]
> Delete blank lines

Docstrings should end with a blank line. Please follow the [numpydoc reference](https://numpydoc.readthedocs.io/en/latest/format.html).

**Review by Lucia-Fonseca — APPROVED** [review]
Good to go

**Pull request merged**